ddev/ddev
# fix: get composer/git to stop complaining about safe dir

## The Issue

Recent reports of problems like 

```
fatal: detected dubious ownership in repository at '/var/www/html'
To add an exception for this directory, call:

        git config --global --add safe.directory /var/www/html
```

For example,
* https://discord.com/channels/664580571770388500/1194832810129772554


## How This PR Solves The Issue

Add safe directory config to the .gitconfig that we already provide.

## Manual Testing Instructions

In lima or Docker Desktop without mutagen on macOS, do a plain DDEV drupal 11 installation and ddev composer install.

You shouldn't see the warning.

## Automated Testing Overview



## Release/Deployment Notes
